model/src/main/scala/hmda/model/census/CBSATractLookup.scala

```diff
@@ -0,0 +1,53 @@
+package hmda.model.census
+
+import java.io.File
+
+import scala.io.Source
+
+object CBSATractLookup {
+  val values: Seq[CBSATract] = {
+    val lines = Source.fromFile(new File("model/src/main/resources/tract_to_cbsa_2013.csv")).getLines
+    lines.map { line =>
+      val values = line.split('|').map(_.trim)
+      val name = values(0)
+      val metDivName = values(1)
+      val state = values(2)
+      val countyFips = values(3)
+      val county = values(4)
+      val tracts = values(5)
+      val geoIdMsa = values(6)
+      val metDivFp = values(7)
+      val smallCounty = values(8).toInt
+      val stateCode = values(9)
+      val tractDecimal = values(10)
+
+      CBSATract(
+        name,
+        metDivName,
+        state,
+        countyFips,
+        county,
+        tracts,
+        geoIdMsa,
+        metDivFp,
+        smallCounty,
+        stateCode,
+        tractDecimal
+      )
+    }.toSeq
+  }
+}
+
+case class CBSATract(
+  name: String,
+  metDivName: String,
+  state: String,
+  countyFips: String,
+  county: String,
+  tracts: String,
+  geoidMsa: String,
```

Comment:
I found `geoIdMsa` a little easier to read.  With this version (when I saw it used elsewhere) I read it as geoid, which is also a word but different.